nautiluszim/scraper.py

```diff
@@ -402,6 +468,7 @@ def make_html_files(self):
                                 "_id": str(docid).zfill(5),
                                 "ti": document.get("title") or "Unknown?",
                                 "dsc": document.get("description") or "",
+                                "longdsc": document.get("long-description") or "",
```

Comment:
No ; this is for entries in the collection ; it's not ZIM metadata

Reply:
I was doubtful about it when I added it. Thanks , I will change it now.